public/pages/CreateChannel/CreateChannel.tsx

```diff
@@ -144,15 +144,25 @@ export function CreateChannel(props: CreateChannelsProps) {
   });
 
   useEffect(() => {
-    setBreadcrumbs([ BREADCRUMBS.NOTIFICATIONS,
+    const { edit } = props;
+    const breadcrumbs = [
+      BREADCRUMBS.NOTIFICATIONS,
       BREADCRUMBS.CHANNELS,
-      props.edit ? BREADCRUMBS.EDIT_CHANNEL : BREADCRUMBS.CREATE_CHANNEL]);
-    window.scrollTo(0, 0);
+    ];
+    if (edit) {
+      if (getUseUpdatedUx()) {
+        breadcrumbs.push(BREADCRUMBS.EDIT_CHANNEL_DETAILS(name))
+      }
+      breadcrumbs.push(BREADCRUMBS.EDIT_CHANNEL)
```

Comment:
Do we need this if we are showing the name?

Reply:
same comment as https://github.com/opensearch-project/dashboards-notifications/pull/257/files?diff=split&w=0#r1747503523